### ISSUE TITLE
removed constructor in 2.4.0? can not build

### ISSUE THREAD

**Issue by v-jaros** [mpetazzoni/sse.js#57]
Hi, I can not build my project after auto-update to 2.4.0. If I'm watching correctly, you removed constructor, so `var source = new SSE(url, options);` is not possible to call. Was the change really intended?

**Comment by caugner** [comment]
We're seeing the same issue [in our Dependabot PR](https://github.com/mdn/yari/pull/10691), caused by [this constructor call](https://github.com/mdn/yari/blob/fc4da56ffd9e616174cfda3313d8c38f82431d62/client/src/plus/ai-help/use-ai.ts#L664-L674).

```
error TS2351: This expression is not constructable.
  Type 'SSE' has no construct signatures.
```

**Comment by mpetazzoni** [comment]
Sorry about this issue, I've published a 2.4.1 release that brings the constructor back into the type definition. I don't fully understand yet why `tsc` removes it on build – I'll dig into that separately.
